src/index.html

```diff
@@ -1,12 +1,423 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="catalog">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+
+  <body class="catalog__body">
+    <header class="header catalog__header">
+      <a class="header__logo-link" href="#home">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="MOYO logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#apple"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#laptops-&-Computers"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="products">
+      <div class="products__container">
+        <ul class="products__list">
+          <li class="products__item">
```

Comment:
Soo many containers and wrappers here
Let's get rid of them
`section` container with `article` for each card inside will be enough

Reply:
@Anna-Tkachenko I have tried to get rid of many containers, but after fixing two of the tests didn't pass. Can you help? @Anna-Tkachenko

Reply:
@serentinos I'll check it in next review 🐝

Reply:
@serentinos I didn't see any failed tests in the link in PR description. Did you re-deploy the tests?